### PR TITLE
Client Sign Up Form + Confirmation + Contact Us

### PR DESCRIPTION
**Pull request by ejohnst2**

### CODE REVIEW

**Comment by roma219** [comment]
@ejohnst2 There is a conflics with yarn.lock and could you please remove package-lock (we are using yarn as i learned) and .DS_Store

**Comment by roma219** [comment]
@ejohnst2  Take a look at this: https://monterail.github.io/vuelidate/ - makes validating forms super easy, it's a vue mixin & has integration with vuetify (https://vuetifyjs.com/en/components/forms - scroll down to vuelidate)

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "axios": "^0.18.0"
+  }
+}
```
Comment (nicksarafa) [inline package.json]: remove this `package.json`

File: public/src/assets/images/bahasa-flag (1).svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="12" viewBox="0 0 18 12">
+    <g fill="none" fill-rule="evenodd">
+        <path fill="#CD2027" d="M0 0h18v6H0z"/>
+        <path stroke="#979797" stroke-width=".5" d="M.25.25h17.5v11.5H.25z"/>
+    </g>
+</svg>
```
Comment (nicksarafa) [inline public/src/assets/images/bahasa-flag (1).svg]: 💣

File: public/src/components/Public/Index.vue [hunk 3]
```diff
@@ -2,17 +2,22 @@
   <v-app>
     <v-toolbar flat class="elevation-1" app color="secondary" clipped-left>
       <v-toolbar-items class="ml-0">
-        <v-btn flat color="primary" @click="$router.push('/manager')">
+        <v-btn flat color="primary" @click="$router.push('/')">
```
Comment (nicksarafa) [inline public/src/components/Public/Index.vue]: there's no way to get to `/manager` besides typing in `manager`

File: public/src/components/Public/Index.vue [hunk 4]
```diff
@@ -54,3 +59,14 @@
     </v-content>
   </v-app>
 </template>
+
+<script>
+  export default {
+    data: () => ({
+      items: [
+        { title: 'Bahasa' },
+        { title: 'English' }
```
Comment (nicksarafa) [inline public/src/components/Public/Index.vue]: these should eventually be images representing the langs

File: public/src/components/Public/SignUp.vue [hunk 5]
```diff
@@ -9,18 +9,16 @@
...
-              <v-text-field label="Email" flat v-model="form.email" />
-              <v-text-field label="SMS & Call Phone" flat v-model="form.sms" />
-              <v-text-field label="Whatsapp" flat v-model="form.whatsapp" />
-              <v-select flat :items="['Houshold', 'Villa', 'Business']" label="Houshold, Villa or Business" v-model="form.type" />
+            <v-form v-model="valid" ref="form" lazy-validation>
+              <v-text-field for="name" label="Write your Full Name" flat v-model="form.name" :rules="nameRules" required />
+              <v-text-field label="Email" flat v-model="form.email" :rules="emailRules" required />
+              <v-text-field label="SMS" flat v-model="form.sms" :rules="smsRules" required />
+              <v-text-field label="Whatsapp" flat v-model="form.whatsapp" :rules="whatsappRules" required />
+              <v-select flat :items="['Houshold', 'Villa', 'Business']" label="Item" v-model="form.type" :rules="[v => !!v || 'Item is required']"required />
```
Comment (nicksarafa) [inline public/src/components/Public/SignUp.vue]: Houshold is spelled in and we'll have to pass in the strings from i18n

**Comment by nicksarafa** [comment]
these changes are already in there

**Pull request closed without merging**